coops-ui/tests/unit/FilingHistoryList.spec.ts

```diff
@@ -90,11 +90,13 @@ const sampleFilings = [
     'filing': {
       'header': {
         'name': 'changeOfAddress',
-        'date': '2019-01-06T19:22:59.003777+00:00',
+        'date': '2019-04-06T19:22:59.003777+00:00',
         'paymentToken': 7891,
-        'certifiedBy': 'Full Name 3',
-        'filingId': 9871,
-        'availableOnPaperOnly': true
+        'certifiedBy': 'Cameron',
+        'filingId': 9873,
+        'availableOnPaperOnly': false,
+        'effectiveDate': '2019/12/13',
```

Comment:
effectiveDate has a timestamp with it. It would be good if you can have the test updated to reflect the change implemented in the last commit.

Reply:
Absolutely, thank you!